packages/core/src/exchanges/subscription.ts

```diff
@@ -104,14 +116,6 @@ export const subscriptionExchange = ({
     });
   };
 
-  const isSubscriptionOperation = (operation: Operation): boolean => {
```

Comment:
NIT: We can save a few bytes by leaving this in and doing `const isSubscriptionOperation = isSubscriptionOperation 
|| (operation: Operation): boolean => {`

Reply:
I hope [9fc6ede](https://github.com/FormidableLabs/urql/pull/2534/commits/9fc6eded11d38afc32ee6e1147f550dd36b77bcf) is what you meant 😁

Alternatively, I also considered using default parameter assignment, but I assumed you don't wanna use this as it needs to be transpiled for IE? https://caniuse.com/?search=default%20parameters